src/DnnSummit.Data/Services/LocationService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using DnnSummit.Data.Extensions;
+using DnnSummit.Data.Models;
+using DnnSummit.Data.Services.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DnnSummit.Data.Services
+{
+    internal class LocationService : BaseService<TwoSexyContent.Location>, ILocationService
+    {
+        public LocationService()
+            : base("https://www.dnnsummit.org/desktopmodules/2sxc/api/app/DnnSummit2019/Query/", "GetLocations")
+        {
+        }
+
+        public async Task<IEnumerable<Location>> GetAsync()
+        {
+            var locations = await QueryAsync();
+            return locations.Select(x => new Location
+            {
+                Title = x.Title,
+                Description = x.Description,
+                HotelName = x.HotelName,
+                HotelStandardsTitle = x.HotelStandardsTitle,
+                HotelStandard1 = x.HotelStandard1,
```

Comment:
Given rename this variable to HotelStandards we would add some code like this

```
HotelStandards = new[]
{
    x.HotelStandard1,
    x.HotelStandard2
}```

Reply:
New commit pushed to resolve.

Reply:
looks good